02_bubble/sarahdavy1983/main_test.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestMainOut(t *testing.T) {
+
+	main()
+
+	expected := []int{1, 2, 3, 5}
+	actual := []int{1, 2, 3, 5}
+
+	if !reflect.DeepEqual(expected, actual) {
+		t.Errorf("Unexpected output in main()")
+	}
+}
+
+var tests = []struct {
+	name  string
+	input []int
+	want  []int
+}{
+	{"valid", []int{3, 2, 1, 5}, []int{1, 2, 3, 5}},
+	{"same number", []int{5, 5, 5, 5}, []int{5, 5, 5, 5}},
+	{"blank", []int{}, []int{}},
+	{"sorted", []int{1, 2, 3, 4}, []int{1, 2, 3, 4}},
+	{"negative number", []int{1, -5, 6, 9}, []int{-5, 1, 6, 9}},
+}
+
+func TestBubble(t *testing.T) {
+	for _, tc := range tests {
```

Comment:
nitpick: tc could be changed with `test` as you're iterating through a list of `tests` and the singular of `tests` could be `test`
just gives more context to what `tc` is, although this is probablty really pedantic